packages/assets/src/index.ts

```diff
@@ -16,6 +16,7 @@ import InfoCircleLine from "./lib/images/InfoCircleLine";
 import InstagramLogo from "./lib/images/InstagramLogo";
 import NEWMLogo from "./lib/images/NEWMLogo";
 import NEWMLogoSmInverse from "./lib/images/NEWMLogoSmInverse";
+import NEWMMarketplaceLogo from "./lib/images/NewmMarketplaceLogo";
```

Comment:
I'm getting a `Module not found` compile error due to the lower case `Newm` instead of `NEWMMarketplaceLogo`